libnavigation-base/src/main/java/com/mapbox/navigation/base/route/model/RouteOptionsNavigation.kt

```diff
@@ -0,0 +1,29 @@
+package com.mapbox.navigation.base.route.model
+
+import com.google.gson.annotations.SerializedName
+import com.mapbox.geojson.Point
+
+class RouteOptionsNavigation(
```

Comment:
Look comment above:

Shouldn't the builder for configuring the different options live here instead of in `NavigationRoute` (`libdirections-offboard` module)?